app/src/main/java/kr/co/connect/boostcamp/livewhere/di/appModules.kt

```diff
@@ -0,0 +1,13 @@
+package kr.co.connect.boostcamp.livewhere.di
+
+
+val appModules = arrayListOf(
+    apiModule
```

Comment:
`p3;` 쉼표처리는 바로 뒤에 가게 하는 것이 좋아보여요
```suggestion
    apiModule,
```